src/kirby/components/list/list.component.html

```diff
@@ -1,7 +1,7 @@
 <!-- HEADER-->
-<ng-container *ngIf="listHeaderTemplate"> 
+<div class="list-header" *ngIf="listHeaderTemplate">
   <ng-container *ngTemplateOutlet="listHeaderTemplate" ></ng-container>
-</ng-container>
+</div>
 
 <!-- BODY-->
 <ul kirbyInfiniteScroll (scrollEnd)="onLoadOnDemand()" [disabled]="!isLoadOnDemandEnabled">
```

Comment:
Reuse groupedItems:
`<ng-container *ngFor="let section of groupedItems">`

Reply:
Agreed!